launch/tier4_perception_launch/launch/object_recognition/detection/detector/camera_bev_detector.launch.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<launch>
+  <!-- Camera parameters -->
+  <arg name="input/camera0/image"/>
+  <arg name="input/camera0/info"/>
+  <arg name="input/camera1/image"/>
+  <arg name="input/camera1/info"/>
+  <arg name="input/camera2/image"/>
+  <arg name="input/camera2/info"/>
+  <arg name="input/camera3/image"/>
+  <arg name="input/camera3/info"/>
+  <arg name="input/camera4/image"/>
+  <arg name="input/camera4/info"/>
+  <arg name="input/camera5/image"/>
+  <arg name="input/camera5/info"/>
+  <arg name="input/camera6/image"/>
+  <arg name="input/camera6/info"/>
+  <arg name="input/camera7/image"/>
+  <arg name="input/camera7/info"/>
+  <arg name="number_of_cameras"/>
+
+  <!-- Model parameters -->
+  <arg name="bev_detection_model" default="bevdet" description="options: `bevdet`"/>
+  <arg name="data_path" default="$(env HOME)/autoware_data/autoware_tensorrt_bevdet" description="packages data and artifacts directory path"/>
+  <arg name="onnx_file" default="$(var data_path)/bevdet_one_lt_d.onnx" description="onnx model file path"/>
+  <arg name="engine_file" default="$(var data_path)/bevdet_one_lt_d.engine" description="engine model file path"/>
```

Comment:
Please do NOT write those implementation-specific variables within this `tier4_perception_launch`.  Following what `transfusion` and `centerpoint` do in their packages, I would recommend to do something like
```
        <arg name="model_path" value="$(var bevdet_model_path)"/>
```

Reply:
Thanks for your suggestion.
I have always been very confused about the setting of `model_path`, it is too cumbersome. 
If referring to the `lidar_centerpoint` node, I should modify it like this:
```
<arg name="data_path" default="$(env HOME)/autoware_data" description="packages data and artifacts directory path"/>
<arg name="bevdet_model_name" default="bevdet_one_lt_d" description="bedet model file name"/>
<arg name="bevdet_model_path" default="$(var data_path)/autoware_tensorrt_bevdet" />
```

Reply:
Thank you for the response (and also sorry for my late response). Yes, that's my intention

> I have always been very confused about the setting of model_path, it is too cumbersome.

I still believe that isolating package-specific parameters from `tier4_perception_launch` would be beneficial for you as well. 

Consider a case when you want to change the engine file name in the near future. With the current implementation, you would need to modify `tier4_perception_launch`, which would require a careful review by maintainers since this is a core package and may critically affect other pipelines as well, resulting in an increased reviewing cost. Instead, by keeping this package-specific implementation within `tensorrt_bevdet`, you would be able to merge without any approval from the maintainers of `tier4_perception_launch`.